testsuite/python/array_properties.py

```diff
@@ -268,12 +266,17 @@ def test_rot_aniso(self):
         self.set_copy(self.system.part[0].gamma_rot)
 
     def test_lb(self):
+        lbf = lb.LBFluid(agrid=0.5, dens=1, visc=1, tau=0.01)
+        self.system.actors.add(lbf)
+
         # Check for exception for various operators
         # LB
-        self.locked_operators(self.lbf[0, 0, 0].velocity)
-        self.locked_operators(self.lbf[0, 0, 0].stress)
-        self.locked_operators(self.lbf[0, 0, 0].stress_neq)
-        self.locked_operators(self.lbf[0, 0, 0].population)
+        self.locked_operators(lbf[0, 0, 0].velocity)
+        self.locked_operators(lbf[0, 0, 0].stress)
+        self.locked_operators(lbf[0, 0, 0].stress_neq)
+        self.locked_operators(lbf[0, 0, 0].population)
+
+        self.system.actors.clear()
```

Comment:
in general this should go into the special function ```tearDown```

Reply:
The actors.clear() is necessary there because otherwise it would interfere with test_partial_periodic(). The latter test raises a C++ error as long as an LB actor is active.

Reply:
I suppose the test_lb() test should go into its own testcase then, because it does not share the same setup with the other tests?

Reply:
Discussed this with Jean-Noel. Agreed. Will do that.

Reply:
Maybe to clarify: the ```tearDown``` method is called automatically after each function that matches ```test*```. See [unittest documentation](https://docs.python.org/3.8/library/unittest.html#unittest.TestCase.tearDown).

Reply:
Yep, I first wrongly assumed it would only be called at the very end of all tests.